website/pages/docs/plugins/sources/aws/configuration.mdx

```diff
@@ -81,7 +81,7 @@ This is the (nested) spec used by the AWS source plugin.
 
 - `scheduler` (string) (default: `dfs`):
 
-  The scheduler to use when determining the priority of resources to sync. Currently, the only supported values are `dfs` (depth-first search) and `round-robin`. This is an experimental feature, and may be removed in the future. For more information about this, see [performance tuning](/docs/advanced-topics/performance-tuning).
+  The scheduler to use when determining the priority of resources to sync. Currently, the only supported values are `dfs` (depth-first search), `round-robin` and `shuffle`. For more information about this, see [performance tuning](/docs/advanced-topics/performance-tuning).
```

Comment:
Add the info to corresponding `_configuration.mdx` file as well?

Reply:
@candiduslynx currently it doesn't have that option specified

```yaml copy
kind: source
spec:
  # Source spec section
  name: aws
  path: cloudquery/aws
  version: "VERSION_SOURCE_AWS"
  tables: ["aws_ec2_instances"]
  destinations: ["DESTINATION_NAME"]
  spec:
    # Optional parameters
    # regions: []
    # accounts: []
    # org: nil
    # concurrency: 50000
    # initialization_concurrency: 4
    # aws_debug: false
    # max_retries: 10
    # max_backoff: 30
    # custom_endpoint_url: ""
    # custom_endpoint_hostname_immutable: nil # required when custom_endpoint_url is set
    # custom_endpoint_partition_id: "" # required when custom_endpoint_url is set
    # custom_endpoint_signing_region: "" # required when custom_endpoint_url is set
    # use_paid_apis: false
    # table_options: nil
```

Reply:
I added it to `_configuration.mdx` here now; it was missing because it's a new option